src/main/java/org/myrobotlab/service/LcdDisplay.java

```diff
@@ -0,0 +1,232 @@
+package org.myrobotlab.service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.myrobotlab.framework.Service;
+import org.myrobotlab.framework.ServiceType;
+import org.myrobotlab.logging.Level;
+import org.myrobotlab.logging.LoggerFactory;
+import org.myrobotlab.logging.LoggingFactory;
+import org.slf4j.Logger;
+
+/**
+ * 
+ * LcdDisplay - Driver to play with lcd display panel
+ * Tested with HD44780 1602 panel, attached to pcf8574
+ * Driver pasted from Poduzov : https://github.com/Poduzov/PI4J-I2C-LCD
+ * 
+ * @author Moz4r
+ * 
+ */
+public class LcdDisplay extends Service {
+
+  public LcdDisplay(String reservedKey) {
+    super(reservedKey);
+    setReady(false);
+  }
+
+  public final static Logger log = LoggerFactory.getLogger(LcdDisplay.class);
+  private static final long serialVersionUID = 1L;
+
+  private Pcf8574 i2cControler;
+

```

Comment:
this probably should be an org.myrobotlab.service.interfaces.I2CController not a Pcf8574 ?

Reply:
Pcf8574 is not a I2CController but IO expander for i2c bus.
I will rename some stuff because I used "i2cControler" as var name..